.tekton/buildah-pull-request.yaml

```diff
@@ -7,13 +7,10 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" &&
-      (".tekton/build-pipeline.yaml".pathChanged() ||
-      ".tekton/buildah-pull-request.yaml".pathChanged() ||
-      ".tekton/buildah-push.yaml".pathChanged() ||
-      "buildah".pathChanged() ||
-      "image_build".pathChanged() ||
-      "Containerfile.image_build".pathChanged())
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "main" && (".tekton/build-pipeline.yaml".pathChanged() || ".tekton/buildah-pull-request.yaml".pathChanged()
+      || ".tekton/buildah-push.yaml".pathChanged() || "buildah".pathChanged() || "image_build".pathChanged()
+      || "Containerfile.image_build".pathChanged())
```

Comment:
Maybe converting it to YAML literal scalar string with `|` can avoid such change by `sigs.k8s.io/yaml`.